doc/install.md

```diff
@@ -0,0 +1,282 @@
+# Installing
+
+## Quickstart
+
+The fastest way to install PyGMT is with the [mamba](https://mamba.readthedocs.io/en/latest/)
+or [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/index.html)
+package manager which takes care of setting up a virtual environment, as well as the
+installation of GMT and all the dependencies PyGMT depends on:
+
+:::: {tab-set}
+::: {tab-item} mamba
+:sync: mamba
+```
+mamba create --name pygmt --channel conda-forge pygmt
+```
+:::
+
+::: {tab-item} conda
+:sync: conda
+```
+conda create --name pygmt --channel conda-forge pygmt
+```
+:::
+::::
+
+To activate the virtual environment, you can do:
+
+:::: {tab-set}
+::: {tab-item} mamba
+:sync: mamba
+```
+mamba activate pygmt
+```
+:::
+
+::: {tab-item} conda
+:sync: conda
+```
+conda activate pygmt
+```
+:::
+::::
+
+After this, check that everything works by running the following in a Python interpreter
+(e.g., in a Jupyter notebook):
+```python
+import pygmt
+pygmt.show_versions()
+```
+
+You are now ready to make you first figure! Start by looking at the tutorials on our
+sidebar, good luck!
+
+:::{note}
+The sections below provide more detailed, step by step instructions to install and test
+PyGMT for those who may have a slightly different setup or want to install the latest
+development version.
+:::
+
+## Which Python?
+
+PyGMT is tested to run on Python {{ requires_python }}.
+
+We recommend using the [Miniforge](https://github.com/conda-forge/miniforge#miniforge3)
+Python distribution to ensure you have all dependencies installed and
+the [mamba](https://mamba.readthedocs.io/en/stable/user_guide/mamba.html) package manager
+in the base environment. Installing Miniforge does not require administrative rights to
+your computer and doesn't interfere with any other Python installations on your system.
+
+## Which GMT?
```

Comment:
The Markdown file is equivalent to the old ReST file, except that I changed:

> PyGMT requires Generic Mapping Tools (GMT) >=6.3.0 since there are many changes being made to GMT itself in response to the development of PyGMT, mainly the new [modern execution mode](https://docs.generic-mapping-tools.org/6.5/cookbook/introduction.html#modern-and-classic-mode).

to 

> PyGMT requires Generic Mapping Tools (GMT) >=6.3.0 since there are many changes being made to GMT itself in response to the development of PyGMT.

Reply:
The new link is at https://docs.generic-mapping-tools.org/6.5/reference/introduction.html#modern-and-classic-mode, but yeah, should be ok to remove this part since GMT 6 has been around for 3+ years now.